### PR TITLE
Update coscheduling doc and add PostFilter

### PR DESCRIPTION
**Pull request by cwdsuzhou**
Fix https://github.com/kubernetes-sigs/scheduler-plugins/issues/244

Update docs.

### CODE REVIEW

File: kep/42-podgroup-coscheduling/README.md [hunk 1]
```diff
@@ -114,6 +115,12 @@ This extension pre-filters pods to save scheduling cycles. This is especially he
 
 For any pod that gets rejected, their pod group would be added to a backoff list and get retried until a TTL is met.
 
+#### PostFilter
+
+This extension will try to reject a group of pods if anyone of the group fails when `filter`
+
+> If more than 99% of pods pass `filter`, we will wait, otherwise we will reject the whole group.
```
Comment (Huang-Wei) [inline kep/42-podgroup-coscheduling/README.md:122]: Does it read better?

```
If the gap to reach the quorum of a PodGroup is greater than 10%, 
we reject the whole PodGroup. Note that this plugin should be configured
as the last one of all PostFilter plugins.
```

**Review by Huang-Wei — COMMENTED** [review]
Thanks @cwdsuzhou !

Can you also update `doc/install.md` to include the `PostFilter` configuration piece?

Also, there is a typo in `PostFilter` logic: notAssiginedPercentage -> notAssignedPercentage.

**Comment by cwdsuzhou** [comment]
> Thanks @cwdsuzhou !
> 
> Can you also update `doc/install.md` to include the `PostFilter` configuration piece?
> 
> Also, there is a typo in `PostFilter` logic: notAssiginedPercentage -> notAssignedPercentage.

Of course, will do.

File: kep/42-podgroup-coscheduling/README.md (hunk 1 above)
Comment (cwdsuzhou) [reply kep/42-podgroup-coscheduling/README.md]: updated

Comment (Huang-Wei) [reply kep/42-podgroup-coscheduling/README.md]: I meant to replace the whole section with the above stanza. (say don't need to specify "This extension ...").

Comment (cwdsuzhou) [reply kep/42-podgroup-coscheduling/README.md]: done

**Comment by seanmalloy** [comment]
/lgtm
/kind documentation
/kind cleanup

@Huang-Wei please review again when you have some time. Thanks!

**Comment by denkensk** [comment]
I will review it today.

**Pull request merged**